src/crystal/system/unix/sysconf_cpucount.cr

```diff
@@ -0,0 +1,9 @@
+{% skip_file if flag?(:openbsd) || flag?(:freebsd) %}
+
+require "c/unistd"
+
+module Crystal::System
+  def self.cpu_count
+    LibC.sysconf(LibC::SC_NPROCESSORS_ONLN)
```

Comment:
I think this could be part of a separate PR as the return value isn't checked elsewhere. Ex: `Process.times`